src/Microsoft.Diagnostics.Runtime/Native/NativeRuntime.cs

```diff
@@ -261,6 +275,95 @@ private void InitThreads()
             _threads = threads.ToArray();
         }
 
+        public override IEnumerable<ClrException> GetSerializedExceptions()
+        {
+            if (!this.ClrInfo.DacInfo.PlatformAgnosticFileName.Contains("mrt100"))
+                throw new ClrDiagnosticsException("Serialized exceptions are only available for a Native data target");
+
+            if (_sosNativeSerializedExceptionSupport == null)
+                throw new ClrDiagnosticsException("This version of mrt100 is too old.", ClrDiagnosticsException.HR.DataRequestError);
+
+            var exceptionById = new Dictionary<ulong, NativeHeap.NativeException>();
+            ISerializedExceptionEnumerator serializedExceptionEnumerator = _sosNativeSerializedExceptionSupport.GetSerializedExceptions();
+            while (serializedExceptionEnumerator.HasNext())
+            {
+                ISerializedException serializedException = serializedExceptionEnumerator.Next();
+
+                //build the stack frames
+                IList<ClrStackFrame> stackFrames = new List<ClrStackFrame>();
+                ISerializedStackFrameEnumerator serializedStackFrameEnumerator = serializedException.StackFrames;
+                while (serializedStackFrameEnumerator.HasNext())
+                {
+                    ISerializedStackFrame serializedStackFrame = serializedStackFrameEnumerator.Next();
+
+                    NativeModule nm = ((NativeHeap)this.GetHeap()).GetModuleFromAddress(serializedStackFrame.IP);
+                    if (nm == null)
+                    {
+                        Trace.WriteLine(String.Format("Unable to resolve module for IP {0}", serializedStackFrame.IP));
+                        continue;
```

Comment:
We should not continue when we encounter a null module.  This leads to an incorrect stack trace.  Not having the module simply means we can't resolve the symbol name...we should still produce a stack frame to our user...even if the only useful thing in the ClrStackFrame is the ip.